OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -40,6 +44,7 @@
     <Compile Include="OpenRA.Game\ActorInfoTest.cs" />
     <Compile Include="OpenRA.Game\OrderTest.cs" />
     <Compile Include="OpenRA.Game\PlatformTest.cs" />
+    <Compile Include="PathfinderTests.cs" />
```

Comment:
I figure we might want to put this in a ```OpenRA.Mods.Common``` folder for consistency with the other tests.